docs/graphql/manual/guides/integrations/auth0-jwt.rst

```diff
@@ -163,3 +163,42 @@ The configured unauthorized role will be used whenever an access token is not pr
 
 This can be useful for data that you would like anyone to be able to access and can be configured and restricted
 just like any other role.
+
+Sync Users from Auth0
+^^^^^^^^^^^^^^^^^^^^^
+
+Now, that you have created an Auth0 application and can signup/login, you will need a way to sync your users in Postgres as well.
+All you really need is the Auth0 ``user_id`` in something like a "users" table.
+
+Using Auth0 Rules again, add the following rule which will insert a new user everytime someone signs-up.
```

Comment:
```suggestion
Using Auth0 rules again, add the following rule which will insert a new user every time someone signs up.
```

Reply:
I feel `Rules` should be capitalized here to emphasize that it's a feature and not used as a common noun.